src/main/java/appeng/client/gui/widgets/GuiCraftingTree.java

```diff
@@ -24,7 +24,7 @@
 public class GuiCraftingTree {
 
     private final AEBaseGui parent;
-    private final int widgetX, widgetY, widgetW, widgetH;
+    public int widgetX, widgetY, widgetW, widgetH;
```

Comment:
Any reason for public instead of protected?

Reply:
It's resized in GuiCraftConfirm which is a different class